pkg/addons/default/aws_node.go

```diff
@@ -76,6 +76,18 @@ func UpdateAWSNode(rawClient kubernetes.RawClientInterface, region string, plan
 			continue
 		}
 
+		// Leave service account if it exists
+		// to avoid overwriting annotations
+		if resource.GVK.Kind == "ServiceAccount" {
+			_, exists, err := resource.Get()
+			if err != nil {
+				return false, err
+			}
+			if exists {
+				continue
```

Comment:
It'd be consistent to have a log message indicating that it was skipped, somewhat like the "pretend it was replaced" log message for `CustomResourceDefinition` directly above.